arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -663,50 +717,55 @@ class ffmpeg_video_input::priv
     }
 
     // if no metadata from the stream, add a basic metadata item
-    if ( retval.empty() )
+    if( retval.empty() )
     {
-      auto meta = std::make_shared<kwiver::vital::metadata>();
+      auto meta = std::make_shared< kwiver::vital::metadata >();
       set_default_metadata( meta );
 
-      retval.push_back(meta);
+      retval.push_back( meta );
     }
 
     return retval;
   }
 
   // ==================================================================
+
   /*
-  * @brief Loop over all frames to collect metadata
-  */
-  void collect_all_metadata()
+   * @brief Loop over all frames to collect metadata
+   */
+  void
+  collect_all_metadata()
   {
     // is stream open?
-    if ( ! this->is_opened() )
+    if( !this->is_opened() )
     {
-      VITAL_THROW( vital::file_not_read_exception, video_path, "Video not open" );
+      VITAL_THROW( vital::file_not_read_exception, video_path,
+                   "Video not open" );
     }
 
-    if ( !collected_all_metadata )
+    if( !collected_all_metadata )
     {
       std::lock_guard< std::mutex > lock( open_mutex );
 
       auto initial_frame_number = this->frame_number();
 
-      if ( !frame_advanced && !end_of_video )
+      if( !frame_advanced && !end_of_video )
       {
         initial_frame_number = 0;
       }
 
       // Add metadata for current frame
-      if ( frame_advanced )
+      if( frame_advanced )
       {
-        this->metadata_map.emplace(this->frame_number(), this->current_metadata());
+        this->metadata_map.emplace(
+             this->frame_number(), this->current_metadata() );
```

Comment:
```suggestion
        this->metadata_map.emplace(
          this->frame_number(), this->current_metadata() );
```

Huh, *this* is an interesting bug. I've seen double indent, but this isn't even aligned to an indent stop.

Happens a couple more times just below. I guess this is in that "should we leave it alone so Uncrustify doesn't keep trying to change it?" category, though I'd really, really love to fix this particular bug.